extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -280,8 +284,8 @@ export default class JupyterServerInstallation {
 
 	private async installSparkMagic(): Promise<void> {
 		if (process.platform === constants.winPlatform) {
-			let sparkMagicPath = path.join(this.extensionPath, 'wheels/sparkmagic-#sparkMagicVersion-py3-none-any.whl'.replace('#sparkMagicVersion', constants.sparkMagicVersion));
-			let installSparkMagic = `${this._pythonExecutable} -m pip install ${sparkMagicPath} --no-warn-script-location`;
+			let sparkWheel = path.join(this._pythonPackageDir, 'sparkmagic-0.12.6.1-py3-none-any.whl');
```

Comment:
Let's keep using `constants.sparkMagicVersion` for the sparkmagic number. Otherwise we have 2 places to update. Long term I'm hoping we can put all this stuff in a requirements.txt and point at a wheelhouse for ones not available online.